page_test.go

```diff
@@ -329,6 +329,22 @@ func (s *S) TestPageScreenshot() {
 	s.Len(kit.Walk(slash("tmp/screenshots/*")).MustList(), 1)
 }
 
+func (s *S) TestFullPageScreenshot() {
+	f := filepath.Join("tmp", kit.RandString(8)+".png")
+	p := s.page.Navigate(srcFile("fixtures/click.html"))
```

Comment:
Check this file `fixtures/scroll.html` under the root dir of the project, this page will be 8000px long and wide. `fixtures/click.html` is too small to test.

Reply:
hmm, I've got an issue with 

```go
p := s.page.Navigate(srcFile("fixtures/scroll.html"))
```

`page.Navigate` not loading the page fully :/ It always returns `600`

Reply:
Also found another thing with js call - on `scroll.html` it doesn't return `8000 x 8000`. Will think about how to tackle that.

Reply:
Check this line:

https://github.com/ysmood/rod/blob/f9774865a0ddedd0914e2cacd7b83678ae398e3d/page_test.go#L318

There's some trick here.

Read the doc of `Element` function, it says it will retry until it finds the element I use it to wait for the page is loaded and rendered. Navigation doesn't make sure the page is fully rendered, since some page may take a long time to load all the images.

If you don't want to use this smart `Element` function, you can do something like:

```go
page.WaitLoad()
```

To actually wait for the page is loaded.

Reply:
`page.WaitLoad()` is literally what I did haha :)